package.json

```diff
@@ -29,31 +29,34 @@
   "author": "Andrew Kelley",
   "license": "BSD-3-Clause",
   "engines": {
-    "node": ">=6"
+    "node": ">=10"
   },
   "browser": "src/browser.js",
   "devDependencies": {
-    "@types/node": "^14.0.1",
-    "espower-loader": "^1.0.0",
-    "intelli-espower-loader": "^1.0.0",
+    "@types/node": "^14.0.14",
+    "acorn": "^7.3.1",
+    "espower-loader": "^1.2.2",
+    "intelli-espower-loader": "^1.0.1",
     "minecraft-wrap": "^1.2.3",
     "mocha": "^8.0.1",
-    "power-assert": "^1.0.0",
+    "power-assert": "^1.6.1",
     "require-self": "^0.2.3",
     "standard": "^14.3.4"
   },
   "dependencies": {
     "yggdrasil": "^1.3.0",
-    "buffer-equal": "^1.0.0",
-    "debug": "^4.1.0",
+    "debug": "^4.1.1",
     "endian-toggle": "^0.0.0",
-    "lodash.get": "^4.1.2",
-    "lodash.merge": "^4.3.0",
+    "lodash.get": "^4.4.2",
+    "lodash.merge": "^4.6.2",
     "minecraft-data": "^2.58.0",
-    "node-rsa": "^0.4.2",
+    "node-rsa": "^1.0.8",
     "prismarine-nbt": "^1.3.0",
     "protodef": "^1.8.0",
-    "readable-stream": "^3.0.6",
-    "uuid-1345": "^1.0.1"
+    "uuid-1345": "^1.0.2"
+  },
+  "optionalDependencies": {
+    "readable-stream": "^3.6.0",
```

Comment:
why is this an optional dependency

Reply:
`readable-stream` resolves problems that it was in ...v0.12? Stream API is stable now and I think code would work even if `readable-stream` package is missing.

Reply:
So why not remove it as a dependency completely then ?

Reply:
I'll remove it later